nodejs/eks/cluster.ts

```diff
@@ -413,12 +413,14 @@ export function createCore(name: string, args: ClusterOptions, parent: pulumi.Co
 
     // Add any requested StorageClasses.
     const storageClasses = args.storageClasses || {};
-    const userStorageClasses: UserStorageClasses = pulumi.output({});
+    let userStorageClasses: UserStorageClasses | undefined;
```

Comment:
This seems odd to me. Would this work instead?

```ts
const userStorageClasses = {} as UserStorageClasses;
```

Reply:
You're right. Great suggestion! Done